eofConvention.fsx

```diff
@@ -0,0 +1,30 @@
+open System.IO
+
+let EolAtEof(path: FileInfo) = 
+
+    use streamReader = new StreamReader (path.FullName)
+    let filetext = streamReader.ReadToEnd()
+    
+    if filetext <> "" then
+        (filetext |> Seq.last = '\n')
+    else
+        true
+
+let NotPrefix (prefix: string) (pathStr: string) =
+    not (pathStr.StartsWith(prefix))
```

Comment:
@realmarv no need for parenteses in the StartsWith call

Reply:
Done